deb/sphinx/DEBIAN/control

```diff
@@ -1,5 +1,5 @@
 Package: sphinx
-Version: 0.3.1
+Version: 0.4.1
```

Comment:
I realize this is already committed, but I'm curious why this is 0.4.1 instead of 0.4.0?